### ISSUE TITLE
document the proper codec and additional_codecs options when used with the http input

### ISSUE THREAD

**Issue by colinsurprenant** [logstash-plugins/logstash-codec-es_bulk#15]
Per issue #12 when using this codec with the http input, the configured codecs in `additional_codecs` are prioritized over the `codec` option and by default `additional_codecs` is set to `{ "application/json" => "json" }` which means that regardless of what you set in the `codec` option `application/json` content type will always be handled by the `json` codec. 

There are 2 ways to solve this;

- clear `additional_codecs` : 
```
input{http{port => 8080 additional_codecs => {} codec => es_bulk}}
```

- set `additional_codecs` : 
```
input{http{port => 8080 additional_codecs => { "application/json" => "es_bulk" } }}
```

This should probably be added to the documentation of the es_bulk codec.

**Comment by karenzone** [comment]
Seems like the explanation for `codec` and `additional_codecs` belongs in `input-http` where they are set.  https://github.com/logstash-plugins/logstash-input-http/pull/104
Questions:
- Do you recommend adding a note to codec-es_bulk doc as well? 
- Anywhere else?

**Comment by colinsurprenant** [comment]
@karenzone I would suggest also adding a note which links to the http input note with something like "be aware of the precedence of additional_codecs over codec if when using the http input with this codec" or something like that ... WDYT?

**Comment by karenzone** [comment]
Fixed with #16 and https://github.com/logstash-plugins/logstash-input-http/pull/104
